corpus/macros.txt

```diff
@@ -63,75 +63,106 @@ a!($a $a:ident $($a);*);
   (expression_statement
     (macro_invocation
       (identifier)
-      (token_tree
-        (identifier))))
+      (macro_invocation_contents
+        (binary_expression
+          (unary_expression
+            (identifier))
+          (MISSING identifier)))))
```

Comment:
This existing test has a lot of "arbitrary token sequences" as macro invocation contents so as expected it gets confused when it tries to parse those as a sequence of expressions